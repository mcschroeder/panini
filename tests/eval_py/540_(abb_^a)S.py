def f540(s: str):
  i = 0
  while i < len(s):
    if s[i] == "a":
      assert s[i+1] == "b"
      assert s[i+2] == "b"
      i += 3
    else:
      i += 1
