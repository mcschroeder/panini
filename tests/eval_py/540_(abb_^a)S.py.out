f540 : {s:𝕊 | s ∈ ([^a]|abb)*} → 𝟙  -- UNVERIFIED (timeout)